src/pages/Education.pug

```diff
@@ -48,7 +48,18 @@ block article
         # Welcome Week
         With the start of the new semester, our university allowed us to present our project and the iGEM competition to the newly arrived international students.
         We got a booth, where we did the DNA extraction as an icebreaker to discuss with the students about our project. This was interesting because most of them do not know much about synthetic biology and working with DNA. 
-        They had a lot of fears, like synthesizing DNA would be dangerous. So we cleared a lot of prejudices up and arouse interest in biology. 
-
+        They had a lot of fears, like synthesizing DNA would be dangerous. So we cleared a lot of prejudices up and arouse interest in biology.
```

Comment:
```suggestion
       Since they did not know a lot about synthetic biology, they had multiple concerns, for example that synthesizing DNA could be dangerous. So we cleared up a lot of prejudices up and arouse interest in biology.
```